src/lesson/lesson9/TableOfUsers/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Table, TableBody, TableCell, TableHead, TableRow } from '@mui/material';
+
+const TableOfUsers = ({ data }) => {
```

Comment:
you may update this component to return another way
`const TableOfUsers = ({ data }) => (
   <div>
     ....
   </div>
)`

Reply:
fixed